docs/index.rst

```diff
@@ -8,13 +8,13 @@ Release v\ |version|. (:ref:`Installation<install>`)
   :alt: Pypi
   :target: https://pypi.python.org/pypi/pytube/
 
-.. image:: https://travis-ci.org/nficano/pytube.svg?branch=master
```

Comment:
we should actually get rid of the travis-ci lines since we aren't using it any more